packages/filecoin-wallet-provider/src/filecoin.ts

```diff
@@ -312,18 +303,19 @@ export class Filecoin {
   getReplaceMessageMinGasParams = async (
     message: LotusMessage,
   ): Promise<{ gasFeeCap: string; gasPremium: string; gasLimit: number }> => {
-    let newFeeCap = message.GasFeeCap
-    const newPremium = new BigNumber(message.GasPremium)
-      .multipliedBy(125)
-      .dividedBy(100)
 
-    if (newPremium.isGreaterThan(message.GasFeeCap)) {
-      newFeeCap = newPremium.toFixed(0, BigNumber.ROUND_CEIL)
-    }
+    const newPremiumBn = new BigNumber(message.GasPremium)
+      .times(1.25)
+      .plus(Number.EPSILON)
+      .integerValue(BigNumber.ROUND_CEIL)
```

Comment:
Let's also leave a small comment here about what we observed error wise that made us bump this up

Reply:
done!